docs/data/scheduler/event-calendar/DefaultPreferences.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { SchedulerEvent } from '@mui/x-scheduler/models';
+import { EventCalendar } from '@mui/x-scheduler/event-calendar';
+import { initialEvents, resources } from '../datasets/personal-agenda';
+
+const defaultPreferences = {
```

Comment:
Nitpick: can you import `EventCalendarPreferences`?
Maybe we can re-export it from `@mui/x-scheduler/event-calendar` 👀